scipy/constants/tests/test_codata.py

```diff
@@ -1,7 +1,5 @@
-from scipy.constants import find, value, ConstantWarning, c, speed_of_light
-from numpy.testing import (assert_equal, assert_, assert_almost_equal,
-                           suppress_warnings)
-import scipy.constants._codata as _cd
+from scipy.constants import constants, codata, find, value, precision
```

Comment:
This is causing the CI failure. We have marked `codata` as deprecated and it will be removed in SciPy 2.0 (I personally don't know about the history here and why, we would need to ask others). This is why in the test it was using `import scipy.constants._codata as _cd`